protocol/messenger_share_urls.go

```diff
@@ -119,23 +123,13 @@ func (m *Messenger) parseCommunityURLWithChatKey(urlData string) (*URLDataRespon
 	}, nil
 }
 
-func (m *Messenger) prepareEncodedRawData(rawData []byte, privateKey *ecdsa.PrivateKey) (string, string, error) {
+func (m *Messenger) prepareEncodedRawData(rawData []byte, privateKey *ecdsa.PrivateKey) (string, error) {
```

Comment:
If there is no signing, there is no sense in `prepareEncodedRawData`, it can be just replaced with `urls.EncodeDataURL(rawData)`